make/test/JtregNativeJdk.gmk

```diff
@@ -78,9 +78,11 @@ ifeq ($(call isTargetOs, windows), true)
   BUILD_JDK_JTREG_LIBRARIES_CFLAGS_libAsyncInvokers := -I$(TEST_LIB_NATIVE_SRC)
 
   BUILD_JDK_JTREG_LIBRARIES_LIBS_libTracePinnedThreads := jvm.lib
+  BUILD_JDK_JTREG_LIBRARIES_LIBS_libNewDirectByteBuffer := -ljava -lc
```

Comment:
This is not correct for Windows

Reply:
Do you have a suggestion?

Reply:
```
BUILD_JDK_JTREG_LIBRARIES_LIBS_libNewDirectByteBuffer := jvm.lib
```
as per the preceding line.

Reply:
Sorry, my bad, that should be:
```
BUILD_JDK_JTREG_LIBRARIES_LIBS_libNewDirectByteBuffer := $(WIN_LIB_JAVA)
```
the Windows equivalent of `-ljava`.

Reply:
Thanks. So changed in 82d9e6f1f6782b180943d1136881d58dd2852e58.